src/node.cc

```diff
@@ -3341,6 +3341,13 @@ void LoadEnvironment(Environment* env) {
   // who do not like how bootstrap_node.js sets up the module system but do
   // like Node's I/O bindings may want to replace 'f' with their own function.
   Local<Value> arg = env->process_object();
+  Local<Object> process_object = env->process_object();
```

Comment:
Would it be an option to change `arg` to type `Local<Object>` or use `dynamic _cast<Local<Object>>(arg)` instead of the new process_object variable?

Reply:
I looked at it when I wrote the changes, but couldn't find a way to convert the handle from `Value` to `Object`. Turns out, casting the objects IN the handles is actually easy. yay!